src/pages/careers/_dept-layout-components/_hero.tsx

```diff
@@ -5,115 +5,139 @@ import Container from './_career-container'
 import { HeaderProps } from './_dept-layout.types'
 import { Header, BackgroundImage } from 'components/elements'
 import device from 'themes/device'
+import { useBrowserResize } from 'components/hooks/use-browser-resize'
 
-const StyledBackGroundImage = styled(BackgroundImage)`
-    height: 660px;
-
-    @media ${device.tablet} {
-        height: 560px;
-    }
-`
-
-const StyledHeader = styled(Header)`
-    margin-bottom: 2.4rem;
-    color: var(--color-white);
-
-    @media ${device.tablet} {
-        text-align: start;
-    }
-`
-
-const Subheadline = styled(Header)`
-    color: var(--color-white);
-    max-width: 99.6rem;
-
-    @media ${device.tablet} {
-        text-align: start;
-    }
-`
-
-const StyledContainer = styled(Container)`
-    max-width: 984px;
-    flex-direction: column;
-    align-items: flex-start;
-    height: 100%;
-
-    @media ${device.tablet} {
-        padding: 0 2rem;
-    }
-`
+type HeaderDataProps = {
+    data: HeaderProps
+}
 
 const query = graphql`
```

Comment:
```suggestion
```
Remove all `graphql` that are touched by this migration. Strictly use `StaticImage`